src/main/java/training/TrainingRobotManager.java

```diff
@@ -1,14 +1,24 @@
 package training;
+import com.ctre.phoenix6.hardware.TalonFX;
+import utils.SmartJoystick;
 
+import training.commands.GotoAngle;
 import utils.DefaultRobotManager;
 
 public class TrainingRobotManager extends DefaultRobotManager {
 
+
+    private TalonFX motor;
+    private SmartJoystick joystick;
+    private final double PI = Math.PI;
```

Comment:
u legit saved PI to not use it and use Math.PI 
![image](https://github.com/user-attachments/assets/3fbcddc8-c42c-4bcd-aec3-c3c676a7c416)